src/visualizer/svg.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * @description SVG functions.
+ * @author henneboy
+ */
+// Inspired by: https://www.w3.org/TR/2011/REC-SVG11-20110816/shapes.html#CircleElement
+const {
+    getLSValuesOfGroup
+} = window.objectFunctions;
+
+(function(){
+    window.svg = {
+        createGroupSvg};
+    // Declaration of global consts, which are layout settings of the svg
+    const svgWidth = window.innerWidth / 2; // width of the svg is half the browser size
+    const svgLineSpace = svgWidth * 0.04; // the amount of space between each line, by the width of the svg
+    const svgLineWidth = svgWidth * 0.01;
+    const svgTextSize = svgLineSpace * 0.5;
+    const RANGEWIDTH = 22;
+
+
+    /**
+     * @summary Creates and appends a div element with the svg of the learningstyles
+     * @param {Array} group the group array with students
+     * @returns {HTMLElement} Html element with the graphical info of the group from the argument
+     */
+    function createGroupSvg(group){
+        let svg = document.createElementNS("http://www.w3.org/2000/svg", "svg");
+        const learningStyles = [];
+        for (let learningStyle in group.students[0].criteria.learningStyles) {
```

Comment:
Hvorfor laver i et array med learningstyles

Reply:
For at man kan tilføje en læringsstil, eller ændre på hvordan de staves/skrives, det er stort set det samme som bare at hardcode et string[] med navnene